.flake8

```
@@ -6,27 +6,23 @@ exclude =
     # https://github.com/PaddlePaddle/Paddle/pull/46290#discussion_r976392010
     ./python/paddle/fluid/[!t]**,
     ./python/paddle/fluid/tra**,
-    # Exclude auto-generated files
-    *_pb2.py,
     # Exclude third-party libraries
     ./python/paddle/utils/gast/**,
     # Exclude files that will be removed in the future, see more at
     # https://github.com/PaddlePaddle/Paddle/pull/46782#issuecomment-1273033731
     ./python/paddle/fluid/tests/unittests/npu/**,
     ./python/paddle/fluid/tests/unittests/mlu/**
 ignore =
-    # E, see https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes
-    E203,
-    E402,
-    E501,
-    E721,E722,E731,E741,
-
-    # F, see https://flake8.pycqa.org/en/latest/user/error-codes.html
-    F405,
-    F841,
-
-    # W, see https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes
-    W503
+    E203, # Whitespace before ‘,’, ‘;’, or ‘:’, it is not compatible with black
+    E402, # Module level import not at top of file
+    E501, # Line too long (82 > 79 characters)
+    E721, # Do not compare types, use `isinstance()`
+    E722, # Do not use bare except, specify exception instead
```

E722 主要是避免 bare except 默认的 `except BaseException` 语义会额外捕获 `KeyboardInterrupt` 和 `SystemExit` 异常，导致用户难以中断

这里不太合适修复主要是因为无法确定到底使用哪个 Exception 替换比较合适，如果捕获范围小了会导致原本应该捕获的异常漏掉，会导致一些严重的错误